packages/driver/src/dom/elements.js

```diff
@@ -363,6 +345,30 @@ const isFocused = (el) => {
   }
 }
 
+const isFocusedOrInFocused = (el) => {
```

Comment:
is this supposed to be `isFocusedOrInFocused` or `isFocusedOrInFocus`, cause currently the method sounds like it is checking if the element is `focused` or if if it an element located as a child of a `focused element.

Reply:
bingo, is focused or inside focused element